### PR TITLE
Remove a lot of text from Git workflow

### PR DESCRIPTION
**Pull request by dlebauer**
I was motivated to just remove the "Basic Workflow" section that starts with "This workflow is for educational purposes only." Then I continued to remove content that I thought was not necessary. A lot of this content dates from previous decades before the carpentries were widespread and before git literacy was common among young scientists.

This is just noise!

image

I tried to begin removing a lot of content that can easily be found on google. 

This section needs a lot of work to be useful - we should make it clear (and consistent) if forking is expected of everyone, or just preferred, etc. Currently there is a focus on branching with a few allusions to forking. And many references to cloning from pecanproject/pecan and branching imply forking isn't the first priority (or when it is). NB: I have a problem with forks because it makes it harder to edit text files in the github web interface.

Could we just start with:

* Use the basic Git Workflow 
* We have a develop branch. No one touches Master except Rob
* Forking is preferred but branching is okay (though old branches at pecanproject may be deleted without notice).
?

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [x] When possible

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: book_source/02_demos_tutorials_workflows/05_developer_workflows/02_git/01_using-git.Rmd [hunk 1]
```diff
@@ -106,41 +119,9 @@ cd pecan
...
-
-5. Before submitting code back to the main repository, make sure that code compiles from the main directory.
-
-`make`
-
-
-6. submit pull request with a reference to related issue; 
-* also see [github documentation](https://help.github.com/articles/using-pull-requests)
-
-
```
Comment (infotroph) [inline book_source/02_demos_tutorials_workflows/05_developer_workflows/02_git/01_using-git.Rmd:140]: 👍 👍 to removing this. It's pretty confusing to lead off with a workflow we don't recommend using.

File: book_source/02_demos_tutorials_workflows/05_developer_workflows/02_git/01_using-git.Rmd [hunk 2]
```diff
@@ -294,19 +265,7 @@ The Rstudio documentation includes useful overviews of [version control](http://
...
-1.  install Rstudio (www.rstudio.com)
-2.  click (upper right) project
-*   create project
-*   version control
-*   Git - clone a project from a Git Repository
-*   paste https://www.github.com/PecanProject/pecan
-*   choose working dir. for repo
-
-#### For development:
+##### Changes without requiring a username and password:
```
Comment (infotroph) [inline book_source/02_demos_tutorials_workflows/05_developer_workflows/02_git/01_using-git.Rmd:268]: You'll need a username whether forking or not -- would it be clearer to say "without membership in the PEcAn organization"? or similar?

**Comment by robkooper** [comment]
One thing that is missing is the recommended (or more required way) to fork PEcAn and start from there. There is a script in the scripts folder called `syncgit.sh` that will keep your fork in sync with the upstream one.

File: book_source/02_demos_tutorials_workflows/05_developer_workflows/02_git/01_using-git.Rmd (hunk 2 above)
Comment (dlebauer) [reply book_source/02_demos_tutorials_workflows/05_developer_workflows/02_git/01_using-git.Rmd:268]: I'll update this to indicate that the forking is the preferred method

**Comment by infotroph** [comment]
@dlebauer This is still marked WIP -- are you planning to do more on it? Seems like a substantial improvement already and I'd vote to merge it now and refine more later.

**Comment by dlebauer** [comment]
@robkooper I've added text to cover syncgit.sh. This should be ready to merge.

**Pull request merged**